pkg/pillar/cmd/volumemgr/updatestatus.go

```diff
@@ -315,13 +316,28 @@ func doUpdateVol(ctx *volumemgrContext, status *types.VolumeStatus) (bool, bool)
 	changed := false
 	switch status.VolumeContentOriginType {
 	case zconfig.VolumeContentOriginType_VCOT_BLANK:
-		// XXX TBD
-		errStr := fmt.Sprintf("doUpdateVol(%s) name %s: Volume content origin type %v is not implemeted yet.",
-			status.Key(), status.DisplayName, status.VolumeContentOriginType)
-		status.SetErrorWithSource(errStr,
-			types.VolumeStatus{}, time.Now())
+		status.State = types.CREATING_VOLUME
+		status.FileLocation = status.PathName()
+		var permission os.FileMode
+		if status.ReadOnly {
+			// RO directory by user, group and others
+			permission = 0444
+		} else {
+			// RW directory by user, group and others
+			permission = 0666
```

Comment:
Should this be `0666`, or should it be more restricted, to `0660` or perhaps even `0600`? What are the security implications? Same for the `0444` above.

Reply:
Isn't this just the permission in dom0? What matters is whether the bind mount is RO or RW.